vars.tf

```diff
@@ -0,0 +1,27 @@
+variable "location" {
+  description = "Azure region"
+  type        = string
+  default     = "eastus"
+}
+
+variable "base_name" {
+  description = "Base name for resources"
+  type        = string
+}
+
+variable "vnet_address_space" {
+  description = "Address space for the virtual network"
+  type        = list(string)
+  default     = ["10.0.0.0/16"]
+}
+
+variable "runner_subnet_address_prefixes" {
+  description = "Address prefixes for the runner subnet"
+  type        = list(string)
+  default     = ["10.0.0.0/24"]
+}
+
+variable "github_org_id" {
```

Comment:
Do you think we should rename this to `github_enterprise_id`?

Reply:
Agreed, done.